### DISCUSSION TITLE
i微信读书浏览器插件——划线、笔记实时同步flomo

### DISCUSSION THREAD

**Discussion by secreter** [flomoapp/3rd-party-tools#68] [分享创造]
## i 微信读书插件
支持[Chrome 应用商店下载](https://chrome.google.com/webstore/detail/i%E5%BE%AE%E4%BF%A1%E8%AF%BB%E4%B9%A6/feicgfcihlfhjjepgibmbieodedbggjh)和[官网下载](https://secreter.github.io/ireader/index.html)。
## 和flomo 协作
在PC 版微信读书中，支持划线、笔记，实时同步到flomo，并且按照书名分类。

![](https://images.redream.cn/upic/2019/20220214203906-2gxtWr.jpg)
![](https://images.redream.cn/upic/2019/20220214204549-vKghrt.jpg)

## 更多功能
为读书爱好者提供最好用的微信读书插件。
目前已支持：
1. 划线句子一键生成图片书签，书签可编辑、下载，支持修改主题和字体。
2. 一键导出全书划线、笔记，格式为Markdown。
3. 可配置，划线句子、批注实时同步到flomo 笔记。
4. 修改背景主题，修改背景纸张材质。
5. 文中图片支持右键复制URL 和图片。
6. 支持文中代码块语法高亮显示、代码复制。
7. 支持查看当前阅读时长和进度。
8. 功能反馈。
![](https://images.redream.cn/upic/2019/20220214204644-STx8mI.jpg)
正在支持:
1. 护眼模式。
2. 修改字体。
3. 划线句子实时同步到飞书、语雀等更多第三方。
4. 代码支持高亮和复制。

**Comment by cmder** [comment]
给的两个链接都失效了

**Comment by subdragon11** [comment]
两个链接都无法访问，请问还会更新么？
